src/bootstrap/flags.rs

```diff
@@ -150,6 +151,7 @@ To learn more about a subcommand, run `./x.py <subcommand> -h`");
             "test"  => {
                 opts.optflag("", "no-fail-fast", "Run all tests regardless of failure");
                 opts.optmulti("", "test-args", "extra arguments", "ARGS");
+                opts.optmulti("", "rustc-args", "compiler arguments", "ARGS");
```

Comment:
maybe we can make the description "extra options to pass the compiler when running tests"

Reply:
:+1:, done!